post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-compute-managing.adoc

```diff
@@ -2,7 +2,26 @@
 :context: multi-architecture-compute-managing
 [id="multi-architecture-compute-managing"]
 = Managing your cluster with multi-architecture compute machines
+include::_attributes/common-attributes.adoc[]
 
-toc::[] 
+toc::[]
+
+== Scheduling workloads on clusters with multi-architecture compute machines
+
+Deploying a workload onto a cluster with compute nodes of different architectures might require further attention in order to successfully place pods in the nodes of your cluster. 
```

Comment:
nit.. remove `might`

Reply:
Does deploying a workload always require further attention, or is this something users won't always have to do manually?

Reply:
sorry..let me rephrase. Adding a 2nd arch to a cluster will always require attention to all new and existing workloads, but not necessarily action. Users have to make sure that everything deployed on their cluster can either run on both architectures, have the proper affinity and/or taints/tolerations, or some other means to guarantee proper scheduling. OCP and kubernetes will not do this for you.

Reply:
Ah I see what you're saying, hmm, what about something like this: 
> Deploying a workload onto a cluster with compute nodes of different architectures requires further attention, and might need further actions, in order to successfully place pods in the nodes of your cluster.

Reply:
@jeffdyoung After the discussion, wdyt of this: 
> Deploying a workload on a cluster with compute nodes of different architectures requires attention and monitoring of your cluster. There might be further actions you need to take in order to successfully place pods in the nodes of your cluster.

Reply:
yeap. Thanks!